be/src/testutil/exprs_test_helper.h

```diff
@@ -172,26 +173,66 @@ class ExprsTestHelper {
     }
 
     static void verify_with_jit(ColumnPtr ptr, Expr* expr, RuntimeState* runtime_state,
-                                const std::function<void(ColumnPtr const&)>& test_func) {
+                                const std::function<void(ColumnPtr const&)>& test_func, bool need_jit = true) {
         // Verify the original result.
         test_func(ptr);
 
+        if (!need_jit) {
+            return;
+        }
         auto jit_engine = JITEngine::get_instance();
         if (!jit_engine->support_jit()) {
             return;
         }
 
         ObjectPool pool;
         auto* jit_expr = JITExpr::create(&pool, expr);
+
         ExprContext exprContext(jit_expr);
         std::vector<ExprContext*> expr_ctxs = {&exprContext};
 
         ASSERT_OK(Expr::prepare(expr_ctxs, runtime_state));
         ASSERT_OK(Expr::open(expr_ctxs, runtime_state));
+        ASSERT_TRUE(jit_expr->is_jit_compiled());
 
-        ptr = expr->evaluate(&exprContext, nullptr);
+        Chunk chunk;
+        chunk.append_column(ptr, 0);
+        ptr = jit_expr->evaluate(&exprContext, &chunk);
         // Verify the result after JIT.
         test_func(ptr);
+
+        Expr::close(expr_ctxs, runtime_state);
+    }
+
+    static void verify_result_with_jit(const ColumnPtr& ptr, Expr* expr, RuntimeState* runtime_state) {
```

Comment:
not sure if this is ok to put into be/src/testutil/ with asserts and checks.

it may be better moved into be/test directory into a new header file. testutil in be/src/ should be modified as little as possible whenever the added code can be moved to be/test.

Reply:
better move to be/test/xxx